### PR TITLE
New landingUi

### PR DESCRIPTION
**Pull request by FizaWaseem**

### CODE REVIEW

File: src/assets/styles/sass/_globalStyles.scss [hunk 1]
```diff
@@ -8,14 +8,15 @@
...
+  background-color: $color-white;
 }
 
 body {
-  background-color: $color-body;
+  background-color: $color-white;
   box-sizing: border-box;
-  width: 128rem;
-  margin: 0 auto;
+  // width: 128rem;
```
Comment (ArchitW) [inline src/assets/styles/sass/_globalStyles.scss]: remove commented code please

File: src/assets/styles/sass/_mixin.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin OrangeButton($bg, $color: white) {
+  background-color: $bg;
+  color: $color;
```
Comment (ArchitW) [inline src/assets/styles/sass/_mixin.scss:9]: from $colpr variable i am not able to guess the actual color. Please rename

File: src/components/UI/LoadingSimple/LoadingSimple.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
+/* eslint-disable prettier/prettier */
 import React from 'react'
 
-import './LoadingSimple.css'
+// import './LoadingSimple.css'
```
Comment (ArchitW) [inline src/components/UI/LoadingSimple/LoadingSimple.js]: remove commented code please

File: src/pages/Landing/Landing.jsx [hunk 4]
```diff
@@ -1,20 +1,100 @@
...
 
 import NavBar from './components/NavBar/NavBar'
 import Footer from './components/Footer/Footer'
 
 import styles from './Landing.module.scss'
 import Hero from './components/Hero/Hero'
+import Course from './components/Course/Course'
+import Places from './components/Places/Places'
+import App from './components/App/App'
+import SimpleSwiper from './components/Sliders/Sliders'
```
Comment (ArchitW) [inline src/pages/Landing/Landing.jsx:12]: can we enable and use [sort-imports](https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports#:~:text=Automatically%20sorts%20imports%20on%20save,Enter%20Sort%20imports)

File: src/pages/Landing/Landing.jsx [hunk 5]
```diff
@@ -1,20 +1,100 @@
...
       <div className={styles.Landing}>
         <NavBar isAuth={false} />
-        <Hero />
-        <div className={styles.backgroundImage} />
-        <div className={styles.backgroundImage2} />
+        <SimpleSwiper />
       </div>
+      {itemList}
+      {/* <Course
+        className="Course"
```
Comment (ArchitW) [inline src/pages/Landing/Landing.jsx]: remove commented code Please

File: src/pages/Landing/components/Footer/Footer.js [hunk 6]
```diff
@@ -1,12 +1,68 @@
...
   return (
     <footer>
-      <CloseIcon className="any-class" />
-      <ArrowDownIcon className="any-class" />
+      <div className={styles.Logo}>
+        <LogoIcon className="LogoIcon" />
+        <p>
+          <CopyRightIcon className="CopyRightIcon" />
+          2021 Fuelup Education
+        </p>
```
Comment (ArchitW) [inline src/pages/Landing/components/Footer/Footer.js:19]: `href="#about"` is placeholder? please use `#` if thats the case

File: src/pages/Landing/components/Hero/Hero.module.scss [hunk 7]
```diff
@@ -1,22 +1,33 @@
...
-    width: 46.6rem;
     text-align: center;
   }
+
   &SecondaryHeading {
     line-height: 3.06;
     letter-spacing: normal;
     text-align: center;
-    color: #b2a9d5;
+    color: #ffffff;
```
Comment (ArchitW) [inline src/pages/Landing/components/Hero/Hero.module.scss]: use variable for color

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 8]
```diff
@@ -1,49 +1,81 @@
...
   left: 0;
+  width: 100%;
   font-family: $font-secondary;
-  height: 4.4rem;
-  display: flex;
-  padding: 3.4rem 4.4rem 0;
-  align-content: center;
-  color: #ffffff;
+  background-color: $primary-color;
+  color: #ffffff !important;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: 1. use a variable for color. 
2. why there is a need to use !important can this be solved with [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 9]
```diff
@@ -1,49 +1,81 @@
...
+  // height: 4.4rem;
+  height: 8.4rem;
+  @include center;
+  // display: flex;
+  padding: .6rem 0rem;
+  /* padding: 3.4rem 4.4rem 0; */
+  // align-content: center;
+  color: #ffffff !important;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: same as above

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 10]
```diff
@@ -1,49 +1,81 @@
...
     display: flex;
-    height: 100%;
     align-items: center;
     list-style: none;
+    color: #ffffff;
+  }
+  &Items {
+    flex: 0 0 58.33333%;
+    max-width: 58.33333%;
+    color: #ffffff;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: same as above

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 11]
```diff
@@ -1,49 +1,81 @@
...
   }
+
   &Item:not(:last-child) {
     padding-right: 3rem;
     line-height: 1.13;
+    color: #ffffff;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: same as above

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 12]
```diff
@@ -1,49 +1,81 @@
...
     }
   }
+
+  &Donate{
+    font-size: 1.5rem;
+    font-weight: normal;
+    line-height: 1.13;
+    text-align: center;
+    padding: .7rem;
+    color: #f16600;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: same as above use variable

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 13]
```diff
@@ -1,49 +1,81 @@
...
+    border-radius: 2rem;
+    border: solid .1rem #f16600;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: use variable

File: src/pages/Landing/components/NavBar/NavBar.module.scss [hunk 14]
```diff
@@ -1,49 +1,81 @@
...
+    background-color: #ffffff;
```
Comment (ArchitW) [inline src/pages/Landing/components/NavBar/NavBar.module.scss]: use variable

File: src/pages/Landing/components/Places/Places.module.scss [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+      width: 60%;
+    }
+  }
+
+  & svg {
+    width: 12px;
+    height: 12px;
+    margin-left: 7px;
+    & path {
+      fill: #fff
```
Comment (ArchitW) [inline src/pages/Landing/components/Places/Places.module.scss]: use variable

**Review by ArchitW — CHANGES_REQUESTED** [review]
Please remove empty files from the PR

**Comment by ArchitW** [comment]
> _No description provided._

it would be nice if you provide few lines for this PR so that review can have some idea of what code they are reviewing

File: src/pages/Landing/Landing.jsx [hunk 16]
```diff
@@ -59,32 +58,8 @@ const Landing = props => {
       <div className={styles.Landing}>
         <NavBar isAuth={false} />
         <SimpleSwiper />
-      </div>
+      </div>g
```
Comment (ArchitW) [inline src/pages/Landing/Landing.jsx]: is this typo? remove g

**Review by ArchitW — CHANGES_REQUESTED** [review]
1 comment please correct.
Rest of PR is LGTM

**Comment by FizaWaseem** [comment]
ok

**Pull request merged**